### DISCUSSION TITLE
How to enable remote debugging

### DISCUSSION THREAD

**Discussion by F404X** [open-wa/wa-automate-nodejs#2639] [Q&A] [answered]
### Are you using the latest version of the library?

- [x] I have checked and am using the latest version of the library.

### What type of session are you experiencing this issue on?

Multi-device and I have set multiDevice to true in my config OR am using the --multi-device flag

### What type of host account are you experiencing this issue on?

Business account

### Mode

EASY API/CLI

### Current Behavior

I insert the debugging-port, the port open, but only white screen appears.
Can someone help me?
Here my configs \\/
![image](https://user-images.githubusercontent.com/73242231/162557098-9bd037c0-df05-4bd3-9f3a-ba401e95d1c2.png)

### create() code

```markdown
const options = {
    sessionId: 'session',
    executablePath: '/usr/bin/google-chrome-stable', // comente esta linha em caso de Windows
	multiDevice: true,
    headless: true,
    qrTimeout: 0,
    authTimeout: 0,
    restartOnCrash: start,
    cacheEnabled: true,
    useChrome: true,
    killProcessOnBrowserClose: true,
    throwErrorOnTosBlock: false,
	waitForRipeSession: true,
    chromiumArgs: [
		'--remote-debugging-port=1313',
        '--no-sandbox',
        '--disable-setuid-sandbox',
        '--aggressive-cache-discard',
        '--disable-cache',
        '--disable-application-cache',
        '--disable-offline-load-stale-cache',
        '--disk-cache-size=0'
    ]
};

wa.create(options)
```

### Environment

```markdown
- OS: Zorin OS 16.1
- Node: 17.9.0
- npm: 8.6.0
```

**Comment by smashah** [comment, marked as answer]
@Foxythefox2020 use `devtool` config instead:

https://docs.openwa.dev/interfaces/api_model_config.ConfigObject.html#devtools

```
const options = {
    sessionId: 'session',
    executablePath: '/usr/bin/google-chrome-stable', // comente esta linha em caso de Windows
	multiDevice: true,
    headless: true,
    qrTimeout: 0,
    authTimeout: 0,
    restartOnCrash: start,
    cacheEnabled: true,
    useChrome: true,
    killProcessOnBrowserClose: true,
    throwErrorOnTosBlock: false,
	waitForRipeSession: true,
	devtools: true
};
```

https://github.com/open-wa/wa-automate-nodejs/issues/122#issuecomment-586580995

**Reply by F404X** [reply]
> @Foxythefox2020 use `devtool` config instead:
> 
> https://docs.openwa.dev/interfaces/api_model_config.ConfigObject.html#devtools
> 
> ```
> const options = {
>     sessionId: 'session',
>     executablePath: '/usr/bin/google-chrome-stable', // comente esta linha em caso de Windows
> 	multiDevice: true,
>     headless: true,
>     qrTimeout: 0,
>     authTimeout: 0,
>     restartOnCrash: start,
>     cacheEnabled: true,
>     useChrome: true,
>     killProcessOnBrowserClose: true,
>     throwErrorOnTosBlock: false,
> 	waitForRipeSession: true,
> 	devtools: true
> };
> ```
> 
> [#122 (comment)](https://github.com/open-wa/wa-automate-nodejs/issues/122#issuecomment-586580995)

I set devtools to true but it keeps starting a blank page

**Reply by smashah** [reply]
Can you paste all of your logs please

**Reply by F404X** [reply]
> ✓ Version: 4.35.3
> ✓ Found session data file: /mnt/AY/Ayumi Yoshida/session.data.json
> ✓ Existing session data injected
> ✓ Page loaded
> ┌─────────────────────┬──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
> │       (index)       │                                                                    Values                                                                    │
> ├─────────────────────┼──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
> │     WA_VERSION      │                                                                  '2.2212.5'                                                                  │
> │       PAGE_UA       │ 'WhatsApp/2.2147.16 Mozilla/5.0 (Macintosh; Intel Mac OS X 11_2_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36' │
> │ WA_AUTOMATE_VERSION │                                                                   '4.35.3'                                                                   │
> │   BROWSER_VERSION   │                                                        'HeadlessChrome/100.0.4896.75'                                                        │
> │         OS          │                                                                 'Linux 5.13'                                                                 │
> │      START_TS       │                                                                1649817962300                                                                 │
> └─────────────────────┴──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
> ✓ Use this easy pre-filled link to report an issue: https://github.com/open-wa/wa-automate-nodejs/issues/new?template=bug_report.yaml&d_info=%7B%0A%20%20%22WA_VERSION%22:%20%222.2212.5%22,%0A%20%20%22WA_AUTOMATE_VERSION%22:%20%224.35.3%22,%0A%20%20%22BROWSER_VERSION%22:%20%22HeadlessChrome/100.0.4896.75%22,%0A%20%20%22START_TS%22:%201649817962300,%0A%20%20%22LATEST_VERSION%22:%20true,%0A%20%20%22CLI%22:%20false%0A%7D&enviro=-%20OS:%20Linux%205.13%0A-%20Node:%2017.9.0%0A-%20npm:%208.5.5&labels=MD
> ✓ Launch inject: 338ms
> ✓ Authenticated
> ✓ Session ready for injection
> ✓ Launch inject: 218ms
> ✓ Launch inject: 355ms
> ✓ WAPI Reinjected
> ✓ Client is ready
> ✓ Downloaded patches in 0.29s
> ✓ Patches Installed: 05676
> ✓ Client loaded for business account [MD] with 598 contacts, 9 chats & 55 messages in 14.746s
> ┌───────────────────────────────────────────────────────┐
> │                                                       │
> │   Use the link below to easily report issues:👇👇👇   │
> │                                                       │
> └───────────────────────────────────────────────────────┘
> ✓ https://github.com/open-wa/wa-automate-nodejs/issues/new?template=bug_report.yaml&d_info=%7B%0A%20%20%22WA_VERSION%22:%20%222.2212.5%22,%0A%20%20%22WA_AUTOMATE_VERSION%22:%20%224.35.3%22,%0A%20%20%22BROWSER_VERSION%22:%20%22HeadlessChrome/100.0.4896.75%22,%0A%20%20%22START_TS%22:%201649817962300,%0A%20%20%22LATEST_VERSION%22:%20true,%0A%20%20%22CLI%22:%20false,%0A%20%20%22LAUNCH_TIME_MS%22:%2014746,%0A%20%20%22ACC_TYPE%22:%20%22BUSINESS%22,%0A%20%20%22chats%22:%209,%0A%20%20%22messages%22:%2055,%0A%20%20%22contacts%22:%20598,%0A%20%20%22isBiz%22:%20true,%0A%20%20%22isMd%22:%20true,%0A%20%20%22INSTANCE_ID%22:%20%22a3d11628-28c4-4594-a8da-a0b1d83f5c2a%22%0A%7D&enviro=-%20OS:%20Linux%205.13%0A-%20Node:%2017.9.0%0A-%20npm:%208.5.5&labels=MD,BHA
> ✓ 🚀 @OPEN-WA ready for account: 2383
